firestore-send-email/functions/package.json

```diff
@@ -18,7 +18,7 @@
     "@types/node": "^12.6.9",
     "@types/nodemailer": "^6.2.1",
     "firebase-admin": "^9.3.0",
-    "firebase-functions": "^3.16",
+    "firebase-functions": "^3.20",
```

Comment:
This might not actually be necessary in retrospect - the outdated version was causing problems when the Emulator Suite was trying to treat this like CF3 functions, but I suspect it won't matter now that this is actually being treated as an extension.